Dockerfile

```diff
@@ -1,14 +1,31 @@
 FROM node:12-alpine
 WORKDIR /usr/src/app
 
-COPY . .
+ENV NODE_ENV=development \
+    PORT=3000 \
+    NEO4J_URL= \
+    NEO4J_USERNAME= \
+    NEO4J_PASSWORD= \
+    JWT_AUTH_KEY= \
+    FILES_S3_BUCKET=
```

Comment:
Shouldn't env vars be defined per env?

Reply:
The env vars here are simply to show that these are the environment variables required in order for this Docker container to run. Technically they are defined on a per-env level so I could remove this. What do you think?

Reply:
Ah they are defaults. Alright that works for me.

Reply:
Sweet!